Monika After Story/game/script-topics.rpy

```diff
@@ -7591,6 +7591,127 @@ label monika_cartravel:
     m 1hua "Just the idea of travelling around the world makes me so excited!"
     m 1eka "I really can't wait, [player]~"
     return
+    
+    ###poems archives
+    
+    default persistent._mas_pm_ask_poems = None #first time player asks
```

Comment:
So, you made a persistent var here, but I don't see you having set it anywhere, nor do I think it'll really be used anywhere else as it doesn't really contribute to the player model.